dawn/src/dawn/Optimizer/PassStageReordering.cpp

```diff
@@ -56,7 +56,7 @@ bool PassStageReordering::run(
     auto newStencil = strategy->reorder(stencilInstantiation.get(), stencilPtr, context_);
 
     stencilInstantiation->getIIR()->replace(stencilPtr, newStencil, stencilInstantiation->getIIR());
-
+    // TODO why is this pointer still used? (it was replaced in the previous statement)
```

Comment:
is this TODO intended? This comment has no code changes around

Reply:
It is a general TODO, it refers to the statement below: stencil pointed by `stencilPtr` has been replaced in the `IIR`, but it's `update` method is called anyways . What do you think?